core/src/main/java/hudson/util/RingBufferLogHandler.java

```diff
@@ -54,6 +54,10 @@ public RingBufferLogHandler(int ringSize) {
         records = new LogRecord[ringSize];
     }
 
+    public static int getDefaultRingBufferSize() {
```

Comment:
Lacks either `@Restricted` or `@since`.

Reply:
Reading the ticket I don't think this should be restricted. The original intent was for plugins to be able to read the value. If it is restricted then that is not possible.